src/app/components/notification/notification.component.ts

```diff
@@ -34,7 +36,20 @@ export class NotificationComponent implements OnInit {
   }
 
   changeAccess(user: User) {
-    this.notificationService.acceptRequestedAccessUser(user.id, user.requested_access_level).subscribe((data: any) => {
+    this.notificationService.acceptRequestedAccessLevelUser(user.id, user.requested_access_level).subscribe((data: any) => {
+      let snackBarRef: MatSnackBarRef<SimpleSnackBar> = this.snackBar.open(`Change ${user.name}'s access level to ${user.requested_access_level}`,
```

Comment:
Shouldn't this be "Changed ..." and not "Change ..."?

Reply:
It represents the action of changing the access level